.github/enos-run-matrices/ent.json

```diff
@@ -1,23 +1,23 @@
 {
   "include": [
     {
-      "scenario": "smoke backend:consul consul_version:1.12.3 distro:ubuntu seal:awskms arch:amd64 builder:crt edition:ent",
+      "scenario": "smoke backend:consul consul_version:1.12.3 distro:ubuntu seal:awskms arch:amd64 build:crt edition:ent",
       "aws_region": "us-west-1"
     },
     {
-      "scenario": "smoke backend:raft consul_version:1.12.3 distro:ubuntu seal:shamir arch:amd64 builder:crt edition:ent",
+      "scenario": "smoke backend:raft consul_version:1.12.3 distro:ubuntu seal:shamir arch:amd64 build:crt edition:ent",
       "aws_region": "us-west-2"
     },
     {
-      "scenario": "upgrade backend:raft consul_version:1.11.7 distro:rhel seal:shamir arch:amd64 builder:crt edition:ent",
+      "scenario": "upgrade backend:raft consul_version:1.11.7 distro:rhel seal:shamir arch:amd64 build:crt edition:ent",
       "aws_region": "us-west-1"
     },
     {
-      "scenario": "upgrade backend:consul consul_version:1.11.7 distro:rhel seal:awskms arch:amd64 builder:crt edition:ent",
+      "scenario": "upgrade backend:consul consul_version:1.11.7 distro:rhel seal:awskms arch:amd64 build:crt edition:ent",
```

Comment:
I dunno, I think I like the old way better. It communicates that we're choosing _how_ something is going to be built, rather than _which_ build we're choosing. But there's the problem: this variant describes both behaviors. If we're going to change it we probably need to think of a better word that communicates `This determines how the vault binary will be built or where it will be provided from`

Reply:
Maybe `artifact_source` or `source`?

Reply:
Curious about your thoughts in particular @rebwill since you're the most articulate of all of us.

Reply:
Hmm. Now that I think about it, the name of the `crt` variant is actually kind of a misnomer, because that variant gets the artifact from our `build` workflow, which lives in the repo and really has nothing to do with CRT other than triggering the CRT workflows when the build workflow runs on merge to main.

What about the following: we call the matrix item `artifact_source`, and the variants are the following: 

* `local_build` -- Build the artifact locally
* `CI_build` -- Receive the artifact from the Build workflow in CI
* `artifactory` -- Receive the artifact from Artifactory

Reply:
Yeah, I chose CRT to convey to people unfamiliar that the build would come _from_ crt. But in both the local and CI build cases we're expecting a local artifact, so I'm confused with the `local`. I'm hesitant to make things too wordy so I'll provide two more suggestions.

`artifact_source:{build_current_branch, local_artifact, download_artifactory}`
`source:{current_branch, local_artifact, artifactory}`

Reply:
Fair. Of those two, I'd go with the first one.

Reply:
Is there a middle ground?

Reply:
the CRT/build confusion has definitely caught me off guard in the past before, I think `artifact_source` is definitely a clearer name, but that then makes `local_artifact` redundant/confusing somewhat? What about `artifact_source:use_local` or similar?

Reply:
`artifact_source:{build,local,artifactory}` ?

Reply:
:( words are hard, in this case I think crt makes better sense to me because I know the artifact is coming from CRT build, while local is it is being built locally.  The branch in both cases will always be current branch, no 😕 ?

Reply:
`artifact_source:{ci,local,artifactory}` ?

Reply:
may be just `source` instead of `artifact_source` keeps it short ?

Reply:
`crt` also builds from the current branch.  I realized the vagueness of `source` when I started writing it, so I updated it to `artifact_source = ["local", "ci", "artifactory"]`.  Does that look like a middle ground :D ?